kytos/utils/config.py

```diff
@@ -147,12 +148,16 @@ def get_remote_metadata(cls):
     @classmethod
     def check_versions(cls):
         """Check if kytos and kytos-utils metadata are compatible."""
-        kytos_metadata = cls.get_remote_metadata()
-        kutils_metadata = cls.get_metadata()
-        kytos_version = kytos_metadata.get('__version__')
-        kutils_version = kutils_metadata.get('__version__')
-
-        if kytos_version != kutils_version:
-            logger = logging.getLogger()
-            logger.warning('kytos (%s) and kytos utils (%s) versions '
-                           'are not equal.', kytos_version, kutils_version)
+        try:
+            kytos_metadata = cls.get_remote_metadata()
+            kytos_version = kytos_metadata.get('__version__')
+        except URLError:
+            pass
```

Comment:
I think we should generate at least a `debug` level log with the connection error.

Reply:
thanks @hdiogenes !